packages/common/src/feedback/_internal/FeedbackBusinessRules.ts

```diff
@@ -0,0 +1,67 @@
+import { IPermissionPolicy } from "../../permissions";
+
+/**
+ * Feedback Permission Policies
+ * These define the requirements any user must meet to perform related actions
+ * @private
+ */
+export const FeedbackPermissions = [
+  "hub:feedback",
+  "hub:feedback:create",
+  "hub:feedback:delete",
+  "hub:feedback:edit",
+  "hub:feedback:view",
+  "hub:feedback:workspace:dashboard",
+  "hub:feedback:workspace:details",
+  "hub:feedback:workspace:settings",
+  "hub:feedback:manage",
+] as const;
+
+/**
+ * Feedback permission policies
+ * @private
+ */
+export const FeedbackPermissionPolicies: IPermissionPolicy[] = [
+  {
+    permission: "hub:feedback",
+    services: ["portal"],
+  },
+  {
+    permission: "hub:feedback:view",
+    dependencies: ["hub:feedback"],
+  },
+  {
+    permission: "hub:feedback:create",
+    authenticated: true,
+    dependencies: ["hub:feedback"],
+    entityEdit: true,
+  },
+  {
+    permission: "hub:feedback:edit",
+    authenticated: true,
+    dependencies: ["hub:feedback"],
+    entityEdit: true,
+  },
+  {
+    permission: "hub:feedback:delete",
+    authenticated: true,
+    dependencies: ["hub:feedback"],
+    entityOwner: true,
+  },
+  {
+    permission: "hub:feedback:workspace:dashboard",
+    dependencies: ["hub:feedback:view"],
```

Comment:
the `:workspace` permissions should likely depend on `hub:feature:workspace` which is the permission connected to the "Workspace Preview" mode... consider looking at how this is done for Sites... where a `hub:site:workspace` permission is defined, which depends on `hub:feature:workspace`, and then all the other workspace permissions depend on `hub:site:workspace`